vql/windows/etw/watch_etw.go

```diff
@@ -78,16 +78,17 @@ func (self WatchETWPlugin) Call(
 			EnableMapInfo: arg.EnableMapInfo,
 		}
 
-		for {
+		for i := 0; i < 10; i++ {
 			err = self.WatchOnce(ctx, scope, arg.Stop, output_chan,
```

Comment:
This needs to be an exponential back-off strategy without giving up.

Reply:
Changed it to a timer that doubles its backoff after each attempt.